packages/lesswrong/components/common/LWHomePosts.tsx

```diff
@@ -529,6 +530,8 @@ const LWHomePosts = ({ children, classes }: {
     [classes.hideOnMobile]: !mobileSettingsVisible,
   });
 
+  const hasSetAnyFilters = filterSettings.tags.length !== 6 || filterSettings.tags.some(setting => setting.filterMode !== "Default")
```

Comment:
Is this going to be true for the default user filters?  I think we have non-"Default" filter modes for some tags by default.

Reply:
Okay, I've got a new line of code here that checks if you have ever changed the tag filters, and then shows it

 const currentFilterSettings = currentUser?.frontpageFilterSettings
 const hasSetAnyFilters = currentFilterSettings === undefined ? false : true;